package com.malware.authservice.exception;

public class UserNotFoundException extends CustomException {

    public UserNotFoundException() {
        super(CustomError.USER_ALREADY_EXISTS.getMessage());
    }
}
